### PR TITLE
Fix/program doc

### PR DESCRIPTION
**Pull request by JiabinYang**
This PR add some comment for `fluid.Program.clone()`

### CODE REVIEW

File: python/paddle/fluid/framework.py [hunk 1]
```diff
@@ -2756,6 +2756,15 @@ class Program(object):
     it will contains nested block.
```
Comment (xsrobin) [inline python/paddle/fluid/framework.py]: contains->contain

File: python/paddle/fluid/framework.py [hunk 2]
```diff
@@ -2756,6 +2756,15 @@ class Program(object):
...
     Please reference the framework.proto for details.
```
Comment (xsrobin) [inline python/paddle/fluid/framework.py:2759]: reference->refer to

File: python/paddle/fluid/framework.py [hunk 3]
```diff
@@ -2756,6 +2756,15 @@ class Program(object):
...
 
+    A set of Program usually contains startup program and main program.
```
Comment (xsrobin) [inline python/paddle/fluid/framework.py]: contains->contain

File: python/paddle/fluid/framework.py (hunk 1 above)
Comment (JiabinYang) [reply python/paddle/fluid/framework.py]: Changed, but these are not related to my PR

File: python/paddle/fluid/framework.py (hunk 2 above)
Comment (JiabinYang) [reply python/paddle/fluid/framework.py:2759]: Changed, but these are not related to my PR

**Pull request merged**